doc/server.md

```diff
@@ -7,21 +7,22 @@ currently supported:
  - `core` (core daemon configuration)
  - `images` (image configuration)
 
-Key                             | Type      | Default   | API extension  | Description
-:--                             | :---      | :------   | :------------  | :----------
-core.https\_address             | string    | -         | -              | Address to bind for the remote API
-core.https\_allowed\_headers    | string    | -         | -              | Access-Control-Allow-Headers http header value
-core.https\_allowed\_methods    | string    | -         | -              | Access-Control-Allow-Methods http header value
-core.https\_allowed\_origin     | string    | -         | -              | Access-Control-Allow-Origin http header value
-core.https\_allowed\_credentials| boolean   | -         | -              | Whether to set Access-Control-Allow-Credentials http header value to "true"
-core.proxy\_http                | string    | -         | -              | http proxy to use, if any (falls back to HTTP\_PROXY environment variable)
-core.proxy\_https               | string    | -         | -              | https proxy to use, if any (falls back to HTTPS\_PROXY environment variable)
-core.proxy\_ignore\_hosts       | string    | -         | -              | hosts which don't need the proxy for use (similar format to NO\_PROXY, e.g. 1.2.3.4,1.2.3.5, falls back to NO\_PROXY environment variable)
-core.trust\_password            | string    | -         | -              | Password to be provided by clients to setup a trust
-images.auto\_update\_cached     | boolean   | true      | -              | Whether to automatically update any image that LXD caches
-images.auto\_update\_interval   | integer   | 6         | -              | Interval in hours at which to look for update to cached images (0 disables it)
-images.compression\_algorithm   | string    | gzip      | -              | Compression algorithm to use for new images (bzip2, gzip, lzma, xz or none)
-images.remote\_cache\_expiry    | integer   | 10        | -              | Number of days after which an unused cached remote image will be flushed
+Key                             | Type      | Default   | API extension           | Description
+:--                             | :---      | :------   | :------------           | :----------
+core.external\_auth.endpoint    | string    | -         | external_authentication | URL of the the external authentication endpoint
```

Comment:
I think it'd probably be cleaner if we moved that to auth.macaroon.endpoint

Reply:
I had this generic so that if we ever plugged another auth type we could have `core.external_auth.protocol: [macaroon|something_else]`

Reply:
Anyway, done